src/api-service/__app__/jobs/__init__.py

```diff
@@ -50,6 +53,31 @@ def post(req: func.HttpRequest) -> func.HttpResponse:
     if isinstance(user_info, Error):
         return not_ok(user_info, context="jobs create")
 
+    # create the job logs container
+    log_container_sas = create_container(
```

Comment:
Does this mean there in only one container named logs and it is shared across all users and all jobs ?

Reply:
yes one container per storage account. create container will randomly assign the storage account if you have more than one

Reply:
does it mean that multiple jobs from multiple users will share same container for their logs ? 

if that's the case - is there  a reason not to separate at least logs from each job into a separate container ?

Reply:
yes multiple job will share the same container. 
No particular reason to not separate.

Reply:
updated to add the job_id as part of the container name
@tevoinea FYI This will change the log paths to 
`{fuzzContainer}/logs-{job_id}/{task_id}/{machine_id}/1.log`